src/GTP.cpp

```diff
@@ -792,6 +795,33 @@ bool GTP::execute(GameState & game, std::string xinput) {
             gtp_fail_printf(id, "syntax not understood");
         }
 
+        return true;
+    } else if (command.find("dump_move_stats") == 0) {
+        std::istringstream cmdstream(command);
+        std::string tmp;
+        int period;
+
+        // tmp will eat dump_move_stats
+        cmdstream >> tmp >> period;
+
+        if (!cmdstream.fail()) {
+            if (period >= 0) {
+                cfg_move_stats = period;
+                gtp_printf(id, "");
+            } else {
+                gtp_fail_printf(id, "invalid period");
+            }
+            if (cfg_allow_pondering) {
+                // Could be send anytime even while pondering
```

Comment:
This will cause the search to be stopped and restarted every time while you send the command. It probably doesn't even work in a non-pondering search?

Reply:
Yes, that's true. But for now lizzie only need that. If another GUI request something else, it will be time to rethink the thing with another merge request :)